angular-ngrx-scss/src/app/shared/components/repo-controls/repo-controls.component.ts

```diff
@@ -135,4 +135,11 @@ export class RepoControlsComponent implements OnInit, OnDestroy {
   handleSortClick(sort: string) {
     this.sortFilter.setValue(sort);
   }
+
+  handleClearClick(): void {
+    this.searchInput.reset();
+    this.typeFilter.setValue(TypeFilter.All);
+    this.languageFilter.setValue(TypeFilter.All);
+    this.sortFilter.setValue(OrderField.UpdatedAt);
```

Comment:
blocker(dry): The initial values here don't need to be explicitly passed. The `reset()` method can be used here instead.

```suggestion
    this.typeFilter.reset();
    this.languageFilter.reset();
    this.sortFilter.reset();
```